scripts/templates/browser-bookmarklet.template.js

```diff
@@ -0,0 +1,293 @@
+const snippet = (name, integrity, version, globalVar, apiKey, userId) => `
+!(function (window, document) {
+  var amplitude = window.${globalVar} || { _q: [], _iq: {} };
+  if (amplitude.invoked) window.console && console.error && console.error('Amplitude snippet has been loaded.');
+  else {
+    amplitude.invoked = true;
+    var as = document.createElement('script');
+    as.type = 'text/javascript';
+    as.integrity = '${integrity}';
+    as.crossOrigin = 'anonymous';
+    as.async = true;
+    as.src = 'https://cdn.amplitude.com/libs/${name}-${version}-min.js.gz';
+    as.onload = function () {
+      if (!window.${globalVar}.runQueuedFunctions) {
+        console.log('[Amplitude] Error: could not load SDK');
+      }
+      window.${globalVar}.init('${apiKey}', '${userId}');
+      const autoTracking = () => {
```

Comment:
@yuhao900914 started publishing snippets for plugins. You could maybe load this too versus including directly in this snippet via template?

Reply:
Thanks for reviewing! Good call, we are planning to build the auto-tracking plugin as a standalone package in later phases, so it can be published following the similar pattern. That requires more review/discussion as it updates the SDK itself, this works like a temp solution. I will update this to use the published one once the plugin is done.

Reply:
I feel there won't require a lot of work. 
Put all things in a plugin package which also allow you to add test. All new plugins are added in the SDK v1.x branch and it's separated from the sdk logic. It's very simple to do, let me know if you need any help on this.

Reply:
Thanks @yuhao900914 ! Sorry for the confusion, my intention was to keep this here and not touch the SDK packages until we are ready to work on the plugin and make it configurable in SDK. In later phases, we can extract this part (probably need add more logic) and refactor into the standalone package, then make it available in the SDK configuration (try to avoid handling the naming, publishing, etc. right now). Plus if I make it a separate package right now, I need to handle the order of [remote loading two scripts](https://jason-ge.medium.com/dynamically-load-javascript-files-in-order-5318ac6bcc61), requiring another refactor of the current loading snippet. Let me know your concerns or suggestions!